core/commands/storage/upload/upload/do_guard.go

```diff
@@ -98,6 +101,27 @@ func doGuard(rss *sessions.RenterSession, res *escrowpb.SignedPayinResult, fileS
 	if err != nil {
 		return fmt.Errorf("failed to send challenge questions to guard: [%v]", err)
 	}
+	go func() {
+		for {
+			tick := time.Tick(10 * time.Second)
+			select {
+			case <-rss.Ctx.Done():
+				return
+			case <-tick:
```

Comment:
you can do time.After because tick is initialized in each loop.. there is no "2nd tick". Unless you want a constant tick every 10s, which can be faster than 10s because of the wait.